bin/cron/applab_datasets/covid19

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env ruby
+
+# This script fetches covid19 data and uploads to the shared firebase channel.
+require_relative '../../../deployment'
+require 'cdo/only_one'
+require 'net/http'
+require 'csv'
+require 'date'
+require 'datapackage'
+require_relative '../../../shared/middleware/helpers/firebase_helper'
+
+US_DATA_URL = "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"
+MAX_DAYS = 90 # there are 55 states/territories, and the table must be less than 5000 total rows (5000/55 = 90.9)
+MAX_WEEKS = 26 # there are 188 countries, and the table must be less than 5000 total rows (5000/188 = 26.5)
+
+def get_us_covid19_data
+  response = Net::HTTP.get_response(URI(US_DATA_URL))
+  return unless response.code == '200'
```

Comment:
should we log to honeybadger if this fails?

Reply:
(you don't need to do this now, but we may want to log failures like this for all live datasets at some point)

Reply:
Yeah, good point we definitely should do some error handling here. https://codedotorg.atlassian.net/browse/STAR-1161 to track